imblearn/over_sampling/_rose.py

```diff
@@ -0,0 +1,121 @@
+"""Class to perform over-sampling using ROSE."""
+
+import numpy as np
+from scipy import sparse
+
+from sklearn.utils import check_random_state
+
+from .base import BaseOverSampler
+from ..utils._validation import _deprecate_positional_args
+
+
+class ROSE(BaseOverSampler):
+
+    """Oversample using Random OverSampling Examples (ROSE) algorithm.
+
+    Read more in the :ref:`User Guide <rose>`.
+    Parameters
+    ----------
```

Comment:
New sections in [restructuredText](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html) need a newline so `sphinx` can build the docs properly.

e.g.

```
Read more ...

Parameters
----------
{sampling_strategy}
```

Reply:
Extra line added.